java/src/main/java/com/google/cloud/dataproc/templates/jdbc/JDBCToGCS.java

```diff
@@ -63,6 +63,10 @@ public void runTemplate() {
       jdbcProperties.put(JDBCOptions.JDBC_NUM_PARTITIONS(), config.getJdbcSQLNumPartitions());
     }
 
+    if (StringUtils.isNotBlank(config.getJdbcFetchSize())) {
```

Comment:
if you are putting a default value against it then "if condition" is not required and can be removed.

Reply:
I've decided not to put a value against the property in template.properties. The JDBC reader has a default of zero so I thought it was fine for me to not set it at all if the user hasn't provided a value.